### PR TITLE
Increase session expiration from 1 to 48 hours

### PR DESCRIPTION
**Pull request by asdil12**
https://progress.opensuse.org/issues/13854

### CODE REVIEW

File: lib/OpenQA/WebAPI.pm [hunk 1]
```diff
@@ -123,6 +123,7 @@ sub startup {
         before_dispatch => sub {
             my $c = shift;
             #$c->app->log->debug(sprintf "this connection is %ssecure", $c->req->is_secure?'':'NOT ');
+            $c->app->sessions->default_expiration(14 * 24 * 60 * 60);
```
Comment (aaannz) [inline lib/OpenQA/WebAPI.pm]: Does this has to be in `begore_dispatch` hook? Isn't it sufficient to put it somewhere in the `startup` routime?

Comment (asdil12) [inline lib/OpenQA/WebAPI.pm]: Yes it is. Patch updated.

**Comment by coolo** [comment]
Sorry, I disagree. 14 days without activity is a lot. IMO it's very fine having to relogin after the weekend. So I would go for 36 hours.

**Comment by asdil12** [comment]
Let's meet at 48 hours.

**Pull request merged**